static/usage/v7/datetime/highlightedDates/array/javascript.md

```diff
@@ -0,0 +1,29 @@
+```html
+<ion-datetime presentation="date" value="2023-01-01"></ion-datetime>
+
+<script>
+  const datetime = document.querySelector('ion-datetime');
```

Comment:
```suggestion
  var datetime = document.querySelector('ion-datetime');
```

Const does not work because of a StackBlitz issue (same change needed for the other JS samples that run in StackBlitz as well)